tests/screen-objects/memphisPropertyPage.js

```diff
@@ -0,0 +1,15 @@
+import BaseHiltonPropertyPage from '/baseHiltonPropertyPage.js'
+
+class MemphisPropertyPage extends BaseHiltonPropertyPage {
```

Comment:
Why would you create a page for Memphis and for NewYork? If I look at the send picture I think you will see that there is a pattern between all cities. What will happen if you have 10 hotels in New York, would you then make New York 1, 2, 3?

How I see it, correct me if I'm wrong, is that the settings page will hold x amount of rooms, you could then collect all the rooms from the app and compare them with the table you provide, this will check:
- the expected rooms
- if you have more or less rooms you will have a failure

Reply:
Their challenge is that they have say 10 properties that they are testing. They are trying to shove everything about every property into a single page object that they are calling say SettingsPage. However, what changes about each property is the number of rooms and the room types. So, it doesn't make sense to have that in a single page object because now they will have a bunch of random properties in the One page. I recommended 2 page objects and the only thing that will change are those values. They will not need more than 2 page objects because the functionality that they are testing will not need to be checked in more than 2 pages. Honestly, it could be done with 1 page, but to keep it safe, I told them to do 2 different ones. The common functionality lives in a base class. The unique functionality lives in a child page. Let me knwo if that makes sense